docs/installation.rst

```diff
@@ -1,5 +1,197 @@
 ============
 Installation
 ============
+Installation documentation for Noggin.
 
-.. note:: **TODO**: Cover end-user installation here.
+
+============
+Requirements
+============
+Noggin requires the following to operate:
+
+1. Access to a FreeIPA/Red Hat Identity Management installation
+2. Access to an Openshift cluster to run the Noggin container
+
+
+===================================
+Install Red Hat Identity Management
+===================================
+For more information see the following documents regarding installation
+
+1. .. _Docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/preparing-the-system-for-ipa-server-installation_installing-identity-management
+2. .. _Playbook install docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/installing-an-identity-management-replica-using-an-ansible-playbook_installing-identity-management
+
+
+Install ansible and ansible-freeipa rpms
+
+::
+    yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+    ARCH=$( /bin/arch )
+    subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"
+    yum install ansible
+    yum install ansible-freeipa
+    yum install yum-plugin-copr
+    yum copr enable ngompa/fedora-aaa
+    yum install freeipa-fas
```

Comment:
`freeipa-fas` has been submitted to EPEL 8: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2020-3f20306084